examples/plotting/histogram.py

```diff
@@ -6,6 +6,7 @@
     :keywords: histogram
 
 '''
+
 import numpy as np
```

Comment:
Existing examples don't have a newline between docstrings and imports
```suggestion
'''
import numpy as np
```

Reply:
Fixed